docs/schema/concepts/by-theme/transportation/index.mdx

```diff
@@ -1,27 +1,140 @@
-- `connector` is a feature type with Point geometry that defines the topology of the transportation network by representing the physical connection between two or more segments. Apart from their Point geometry and the core properties required for all Overture features, connectors do not have any other properties
-
-## Theme concepts
-- **[Roads:](/schema/concepts/by-theme/transportation/roads)** the transportation theme schema models any kind of road, street or
-path, including dedicated walking and cycling paths, as `road` segments Roads are currently the most developed part of the transportation schema. 
-- **[Scoped and rule-based properties:](/schema/concepts/scoping-rules)** the transportation theme schema allows property values to be specified for granular scopes at the sub-feature level. For example:
+
+The transportation schema defines two feature types: 
+
+- The `segment` type has a LineString geometry that represents the center-line of a path that a person or object may travel. Segment properties describe both the physical proprerties (e.g. road surface and width) and non-physical properties (e.g. access restriction rules) of that path. 
+- The `connector` type has a Point geometry that defines the topology of the transportation network by representing the physical connection between two or more segments. Connectors do not have properties beyond geometry and the common properties required for all Overture features.
+
+
+To better support routing use cases, we added to the segment feature type [a new property](https://github.com/OvertureMaps/schema/pull/257) called
+`connectors`: an array of IDs with pre-computed linear reference values that explicitly link segments and
+[connector features](https://docs.overturemaps.org/schema/reference/transportation/connector/) via coordinates. The `connectors` property contains a list of connector features to which a segment feature is physically connected and the relative location of those connector features along the segment. Each connector is a possible routing decision point, meaning it defines a place along the segment in which there is possibility to transition to other segments that share the same connector.
+
+
+## Subtypes
+
+### [Road](/schema/concepts/by-theme/transportation/roads)
+The Overture schema models any kind of road, street or path, including dedicated walking and cycling paths, as `road` segments. Roads are currently the most developed part of the transportation schema.
+
+### Rail
+
+### Water
+
+## Classes
+
+Captures the type of road, railway, or waterway and its position in the transportation network hierarchy. For example, road classes include motorways, primary roads, secondary roads, residential streets, pedestrian pathways, and cycleways. 
```

Comment:
Next heading on "subclasses" begins with "The subclass property defines..." Should this one begin with "The class property captures..."?

(Are we using `code` for property names, *e.g.* `class`?)